### PR TITLE
help: finish pluralizing "the pipeline" term in output

### LINKED ISSUE #2158: status: gives incomplete message when all pipelines are up to date
```console
$ dvc version
DVC version: 0.41.3
Python version: 3.7.3
Platform: Darwin-18.6.0-x86_64-i386-64bit
```
```console
$ dvc pipeline list
train.dvc
featurize.dvc
evaluate.dvc
prepare.dvc
data/data.xml.dvc
================================================================================
foo.dvc
zoo.dvc
================================================================================
2 pipeline(s) total

$ dvc status
Pipeline is up to date. Nothing to reproduce.
```

In this case (having more than one pipeline) the message should be something like `All pipelines are up to date. `.

---
UPDATE: Other commands may have a similar problem, where THE pipeline is referred to in singular. E.g. `metrics modify` has or used to have a `unable o find file '<path>' in the pipeline` message. Should we review all?

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Fix #2158

* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it?

---

Doc update: https://github.com/jorgeorpinel/dvc.org/commit/76f08e8

### CODE REVIEW

File: scripts/completion/dvc.zsh [hunk 1]
```diff
@@ -216,7 +216,7 @@ _dvc_repro=(
   {-R,--recursive}"[Reproduce all stages in the specified directory.]"
   "--ignore-build-cache[Reproduce all descendants of a changed stage even if their direct dependencies didn't change.]"
   "--no-commit[Don't put files/directories into cache.]"
-  "--downstream[Reproduce the pipeline starting from the specified stage.]"
+  "--downstream[Reproduce pipeline(s) starting from the specified stage.]"
```
Comment (shcheklein) [inline scripts/completion/dvc.zsh]: if take a single stage, then it's about reproducing a single pipeline only. If it accepts multiple targets from different pipelines (I would expect so), then we need to change the "specified stage" part here

File: dvc/command/repro.py [hunk 2]
```diff
@@ -152,6 +152,6 @@ def add_parser(subparsers, parent_parser):
         "--downstream",
         action="store_true",
         default=False,
-        help="Reproduce the pipeline starting from the specified stage.",
+        help="Reproduce pipeline(s) starting from the specified stage.",
```
Comment (shcheklein) [inline dvc/command/repro.py]: see my concern about this message below

File: scripts/completion/dvc.zsh (hunk 1 above)
Comment (jorgeorpinel) [reply scripts/completion/dvc.zsh]: True! Should be `stage(s)` indeed. Let me double check and will update.

File: dvc/command/repro.py (hunk 2 above)
Comment (jorgeorpinel) [reply dvc/command/repro.py]: Updated in 6f5449fb

**Comment by jorgeorpinel** [comment]
Done. Should I also review "(s)" in all other command outputs? There's only a handful:

image

**Comment by shcheklein** [comment]
cool stuff! yep, it would be great to fix other obvious place.

**Pull request merged**